pkgs/by-name/om/omada-sdn-controller/package.nix

```diff
@@ -0,0 +1,149 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  writeScriptBin,
+  coreutils,
+  curl,
+  jsvc,
+  openjdk17,
+  mongodb-ce,
+  omada-sdn-controller,
+  testers,
+  writeShellScript,
+  common-updater-scripts,
+  pup,
+  gnugrep,
+  gnused,
+  dataDir ? "/var/lib/omada",
+}:
+
+let
+  # The 'arch' command is required during runtime with is equivalent to 'uname -m':
+  arch = writeScriptBin "arch" "uname -m";
+
+  # Java 17 is supported, pin to avoid unsupported dependency upgrades.
+  jre = openjdk17;
```

Comment:
You mean above?

Reply:
What do you mean? I don't just want to use `jre` as a dependency